### ISSUE TITLE
When manually setting ranges in chart settings, defaults should be min() and max of the series, not 0 and 100

### ISSUE THREAD

**Issue by salsakran** [metabase/metabase#3061]
Right now if I untoggle the "auto y-axis range", it defaults to a min of 0 and a max of 100.

We should probably pre-seed this with min(0,min(series)) and max(series)

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by MarcGJA** [comment]
Good idea, and it would be even better built into the auto-range itself as mentioned here:
https://discourse.metabase.com/t/chart-scaling-woes/7196/3

**Comment by GarretOConnell** [comment]
Is this issue still being worked on, as I would like to see this feature.

**Comment by nodesocket** [comment]
This is a must-have change. auto y-axis should use the min and max values.

**Comment by mazzma12** [comment]
I think the value of this is underestimated.
I don't see a use case where you want your KPIs scaled to zero, min-max scaling should be the default.
The trick I am using at the moment is manually settings min and max values inside the timeseries ranges, and metabase will automatically min max scale.

**Comment by cdeweyx** [comment]
We now offer the ability to "unpin from zero" so when enabled auto y-axis will use the min and max of your series. I believe this addresses the issue here. Please do drop a comment if you have other thoughts. 🙏

<img width="1830" height="641" alt="Image" src="https://github.com/user-attachments/assets/a23f4fc7-ea58-4d9b-a610-442cc9e169f2" />

**Comment by nodesocket** [comment]
Awesome, 👏 been waiting for this feature for years!! 🎊
